src/services/KhlSeason.ts

```diff
@@ -0,0 +1,53 @@
+import { SnokingSeasonResponse } from "../../typings/snokingData";
```

Comment:
I would handle this in a follow up PR, not this one, but we should think about reducing the amount of information in SnokingSeasonResponse to just what we need to do the mappings later; we're never going to need to use stuff like `isRosterSet` as this shows.

Looks good for now, but would be good for us to clean up later.